contracts/test/DeployableERC20Detailed.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.0;
+
+// NOTE:
+//  This file's purpose is to have an ERC20Detailed contract, which is not
+//  just an interface, but an actual contract and can be deployed.
+//
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+
+
+contract DeployableERC20Detailed is ERC20Mintable, ERC20Detailed {
+    constructor(string memory name, string memory symbol, uint8 decimals) public ERC20Detailed(name, symbol, decimals) {}
+}
```

Comment:
Oh yes. I have needed something like this before. I am glad you finally got this working. I might even suggest we use something like this in `dex-liquidity-provision`. Look at my crappy solution, because I wasn't able to get the "double is" working properly...

https://github.com/gnosis/dex-liquidity-provision/blob/master/contracts/test/DetailedMintableToken.sol


Actually, I asked about this probably two weeks ago in the "Problems" section of our daily. Would have avoided double work if this was addressed earlier.

Reply:
Sorry, Ben, I must have missed this request in the Problem sections. Next time, please ping me no one is helping you, although you were raising the concern in the Problem sections